app/components/templates/templates.html

```diff
@@ -195,6 +195,30 @@
               </div>
             </div>
             <!-- !volume-mapping -->
+            <!-- extra-host -->
+            <div class="form-group" >
+              <div class="col-sm-12" style="margin-top: 5px;">
+                <label class="control-label text-left">Hosts file entries</label>
+                <span class="label label-default interactive" style="margin-left: 10px;" ng-click="addExtraHost()">
+                  <i class="fa fa-plus-circle" aria-hidden="true"></i> add additional entry
+                </span>
+              </div>
+              <!-- extra-host-input-list -->
+              <div class="col-sm-12">
+                <div class="col-sm-12 form-inline" style="margin-top: 10px;">
+                  <div ng-repeat="(idx, host) in state.selectedTemplate.Hosts" style="margin-top: 2px;">
```

Comment:
Actually, we need the `track by $index` the in the `ng-repeat` instruction otherwise it creates a strange behavior in the UI when typing in the inputs.

```
<div ng-repeat="(idx, host) in state.selectedTemplate.Hosts track by $index" style="margin-top: 2px;">
```